www-src/app/replicator/replicator_backups.coffee

```diff
@@ -160,6 +160,10 @@ module.exports =
 
             toUpload = []
 
+            # Filter images : keep only the ones from Camera
+            # TODO: Android Specific !
+            images = images.filter (path) -> path.indexOf('/DCIM/') != -1
```

Comment:
True, I just add one.
We may not encounter too much problems. There is a great variety of DCIM subfolders (Camera, ANDRO100, ...), but DCIM looks really like a standard on android.